cmd/console/main.go

```diff
@@ -179,79 +59,11 @@ func main() { // nolint:funlen,gocyclo
 	rootCmd.Flags().BoolVarP(&verbose, "verbose", "v", false, "verbose scan")
 	rootCmd.Flags().BoolVarP(&logFile, "log-file", "l", false, "log to file info.log")
 	rootCmd.Flags().BoolVarP(&generateID, "generate-id", "g", false, "generate uuid for query")
+	rootCmd.Flags().BoolVarP(&version, "version", "", false, "Show kics's current version")
```

Comment:
change to `rootCmd.Flags().BoolVar(&version, "version", false, "display current version")` in order to have a --version flag